repo/packages/A/artifactory-secondary/0/marathon.json.mustache

```diff
@@ -0,0 +1,86 @@
+{
+  "id": "{{artifactory-secondary.framework-name}}",
+  "cpus": {{artifactory-secondary.cpus}},
+  "mem": {{artifactory-secondary.mem}},
+  {{^artifactory-secondary.high-availability.enabled}}
+  "instances": 1,
+  {{/artifactory-secondary.high-availability.enabled}}
+  {{#artifactory-secondary.high-availability.enabled}}
+  "instances": {{artifactory-secondary.high-availability.nodes}},
+  {{/artifactory-secondary.high-availability.enabled}}
+   "container": {
+       "type": "DOCKER",
+       "docker": {
```

Comment:
We should specify the ports required here for the container. This will ensure Mesos will only schedule the container on machines with these ports available (see the requirePorts setting).